shavar/e2e-test/prefs.ini

```diff
@@ -9,20 +9,28 @@ browser.safebrowsing.phishing.enabled = false
 browser.safebrowsing.malware.enabled = false
 privacy.trackingprotection.enabled = true
 browser.safebrowsing.provider.mozilla.nextupdatetime = 1
-browser.safebrowsing.provider.mozilla.gethashURL = https://shavar.stage.mozaws.net/downloads?client=SAFEBROWSING_ID&appver=%VERSION%&pver=2.2
+browser.safebrowsing.provider.mozilla.gethashURL = http://127.0.0.1
```

Comment:
Gethash is unused. If a list is mentioned in `urlclassifier.disallow_completions` then we will not use gethash.

You're right, we can probably just remove that line and just keep the default value.